docker-compose.yaml

```diff
@@ -35,6 +35,7 @@ services:
       - KEYCLOACK_ENABLED=True
       - TENANT_ADMIN_ROLE_ID=9dc79aa8-d42f-4720-8de0-fe79a00a46b7
       - KEYCLOACK_ADMIN_ENDPOINT=http://keycloak:8080/admin/realms/default
+      - OPA_ENDPOINT=http://ext_authz-opa-service:8181
```

Comment:
this needs to be documented

Reply:
also, considering there may be for multiple instances of opa deployment, comma separation support would be nice.